tests/Service/AssetManagerTest.php

```diff
@@ -460,11 +461,15 @@ public function testResponseHeadersForAsset()
         }
 
         $mimeType = $mimeResolver->getMimeType(__FILE__);
+        $lastModified   = new \DateTime();
```

Comment:
Yea but you might need to tell me what's wrong since it's passing CS on build.   Clearly this ones got an extra space but no clue on the other